src/config/cli_config.cpp

```diff
@@ -0,0 +1,155 @@
+#include "cli_config.hpp"
+
+#include <iostream>
+
+#include "cli_tools.hpp"
+
+using namespace std;
+
+namespace bpo = boost::program_options;
+
+namespace taraxa {
+
+CliConfig::CliConfig(int argc, const char* argv[], const std::string& taraxa_version)
+    : main_options("OPTIONS"),
+      node_command_options("NODE COMMAND OPTIONS"),
+      allowed_options("Allowed options"),
+      network_id(DEFAULT_NETWORK_ID) {
+  // Set node as default command
+  command.push_back(NODE_COMMAND);
```

Comment:
might be cleaner to put all this processing from constructor into some separate method